src/app/static/src/components/parameters/EditPhases.vue

```diff
@@ -32,12 +37,23 @@
                   <br/>
                   <div class="phase-start">Start: {{displayPhases[index].start}}</div>
                   <div class="phase-end">End: {{displayPhases[index].end}}</div>
-                  <div class="phase-rt">Rt: {{displayPhases[index].value}}</div>
+                </div>
+                <div class="phase-rt">Rt:
+                  <input
+                    :id="`phase-rt-${index}`"
+                    type="number"
+                    :min="rtMin"
+                    :max="rtMax"
+                    :value="sliderValues[index].value.rt"
+                    step="0.01"
+                    @change="updateRt(index, $event)"
+                    @mousedown.stop="">
```

Comment:
...and this prevents unintended dragging of phases when user selects text with drag in Rt input